modules/spotfleet-cluster/outputs.tf

```diff
@@ -0,0 +1,18 @@
+# ------------------------------------------------------------------------------
+# Output
+# ------------------------------------------------------------------------------
+output "id" {
+  value = "${aws_ecs_cluster.main.id}"
```

Comment:
Outputs should have descriptions. Should more or less be able to copy-paste this from the cluster module.